build.gradle

```diff
@@ -127,11 +128,45 @@ jacocoTestReport {
 
 project.tasks['sonarqube'].dependsOn jacocoTestReport
 
+pitest {
+  group = "Verification"
+  pitestVersion.set('1.5.1')
+  junit5PluginVersion.set('0.12')
+  testSourceSets = [sourceSets.test]
+  mainSourceSets = [sourceSets.main]
```

Comment:
are those needed? these look like defaults

Reply:
yes ,other wise mutator does n't know which files to be mutated.